README.md

```diff
@@ -4,6 +4,7 @@
 </p>
 
 [![All Contributors](https://img.shields.io/badge/all_contributors-7-orange.svg?style=flat-square)](#contributors)
+[![Travis CI](https://travis-ci.com/infinitered/nsfwjs.svg?branch=master)]()
```

Comment:
The image will be available after Travis will be included in the project.

Reply:
Sounds good.  I'll give this a go right now.